frontend/src/util/apiWrapper.js

```diff
@@ -0,0 +1,178 @@
+import { Semaphore } from 'await-semaphore';
+import axios from 'axios';
+import { createDefaultBundleName, pathIsArchive, getArchiveExt } from './worksheet_utils';
+
+export const get = (url, params) => {
+    const requestOptions = {
+        params,
+    };
+    return axios.get(url, requestOptions).then((res) => res.data);
```

Comment:
is there value in returning the `Promise` without resolving it?

Reply:
axios.get is asynchronous so everything needs to be chained with .then() and .catch(), when the axios call is finished it will automatically resolve with response. We were thinking about doing async await for everything but that will probably be later and for a separate ticket.